SAMLPortal/Startup.cs

```diff
@@ -67,25 +70,22 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env, S
 			}
 			else
 			{
-				app.UseExceptionHandler("/Home/Error");
+				//app.UseExceptionHandler("/Error");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)